doc/source/whatsnew/v0.23.0.txt

```diff
@@ -294,7 +294,7 @@ Conversion
 - Bug in :meth:`DatetimeIndex.astype` when converting between timezone aware dtypes, and converting from timezone aware to naive (:issue:`18951`)
 - Bug in :class:`FY5253` where ``datetime`` addition and subtraction incremented incorrectly for dates on the year-end but not normalized to midnight (:issue:`18854`)
 - Bug in :class:`DatetimeIndex` where adding or subtracting an array-like of ``DateOffset`` objects either raised (``np.array``, ``pd.Index``) or broadcast incorrectly (``pd.Series``) (:issue:`18849`)
-
+- Bug in :func:`Series.dt.end_time` where time values in ``Period`` objects were not adjusted (:issue:`17157`)
```

Comment:
and ``PeriodIndex.end_time``

Reply:
we will need a sub-section showing this I think.

Reply:
And if we're making the change in ``to_timestamp(how='end')`` instead of in ``dt.end_time`` (which then calls ``to_timestamp(how='end')``) then I will also include ``to_timestamp(how='end')`` in the sub-section